escalation-eda/escalation-swf/src/main/resources/specs/notifications.yaml

```diff
@@ -1,12 +1,14 @@
 openapi: 3.0.3
```

Comment:
Why not to reply on the notifications spec instead of maintaining its own copy?

Reply:
This is required so that the Sonata Workflow will generate necessary API code to call the notifications.